build/pipeline-releasebuild.yaml

```diff
@@ -46,14 +47,29 @@ steps:
     BuildPlatform: '$(BuildPlatform)'
     BuildConfiguration: '$(BuildConfiguration)'
     MsalClientSemVer: $(MsalClientSemVer)
-    Solution: 'Libs.sln'
+    Solution: 'LibsAndSamples.sln'
 
 # Run Post-build code analysis (e.g. Roslyn)
 - template: template-postbuild-code-analysis.yaml
 
 # Pack and sign all of the nuget packages
 - template: template-pack-and-sign-all-nugets.yaml
 
+- task: UseDotNet@2
+  displayName: 'Use .Net Core SDK 3.x'
+  inputs:
+    version: 3.x
+
+# Run Unit Tests
+- template: template-run-unit-tests.yaml
+  parameters:
+    BuildConfiguration: '$(BuildConfiguration)'
+
+# Run Integration Tests
+- template: template-run-integration-tests.yaml
```

Comment:
What about UI automation?

Reply:
The Win8 tests are not working atm, but should be included.

Reply:
Win 8 and UI auto will be in a separate build that is automatically triggered when this runs. The trigger settings do not show up in the PR but they are visible in the pipeline settings

Reply:
right, but the device auth tests are not running in the VM. there is a separate issue for that.

Reply:
Understood. I am working on fixing it